engine/class_modules/sc_death_knight.cpp

```diff
@@ -4590,6 +4628,34 @@ struct defile_t : public death_and_decay_base_t
   }
 };
 
+struct deaths_due_t : public death_and_decay_base_t
+{
+  deaths_due_t( death_knight_t* p, const std::string& options_str ) :
+    death_and_decay_base_t( p, "deaths_due", p -> covenant.deaths_due )
+  {
+    damage = new deaths_due_damage_t( p );
+
+    parse_options( options_str );
+  }
+
+  void execute() override
+  {
+    debug_cast<deaths_due_damage_t*>( damage ) -> pestilence_procs_per_cast = 0;
+
+    death_and_decay_base_t::execute();
+  }
+
+  bool ready() override
```

Comment:
You should make sure that this only returns true with the covenant selected (spelldata validation should do it automatically) and that death_and_decay_t::ready() returns false for night fae.